sw/device/tests/BUILD

```diff
@@ -2100,10 +2100,8 @@ otp_json(
         otp_partition(
             name = "CREATOR_SW_CFG",
             items = {
-                "CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG": "0000090909",
-                # TODO(#14814): enable flash scrambling / ECC by replacing above
-                # with below.
-                #"CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG": "0000090606",
+                # Enable flash scrambling / ECC by replacing above with below.
```

Comment:
Do you want to update this comment? Looks like "above" is gone.

Reply:
oh yeah, good catch. fixed.